MOBILE_API.xml

```diff
@@ -6020,6 +6024,10 @@
         <param name="myKey" type="VehicleDataResult" mandatory="false">
             <description>Information related to the MyKey feature</description>
         </param>
+        <param name="handsOffSteering" type="VehicleDataResult" mandatory="false" since="6.2">
+            <description>To indicate whether driver hands are off the steering wheel</description>
+        </param>
+
```

Comment:
```suggestion
```

Reply:
@jacobkeeler done in https://github.com/smartdevicelink/rpc_spec/pull/251/commits/0604812c3de6ce7232122d546780e03cb6c2f095